activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -19,7 +19,18 @@ module ActiveSupport
   #   encrypted_data = crypt.encrypt_and_sign('my secret data')                  # => "NlFBTTMwOUV5UlA1QlNEN2xkY2d6eThYWWh..."
   #   crypt.decrypt_and_verify(encrypted_data)                                   # => "my secret data"
   class MessageEncryptor
-    DEFAULT_CIPHER = "aes-256-cbc"
+    class << self
+      #use_authenticated_message_encryption is used to activate 'aes-256-gcm' as default cipher
```

Comment:
This comment doesn't really add anything to me. Let's just document it elsewhere.